data/json/professions.json

```diff
@@ -540,8 +540,8 @@
   },
   {
     "type": "profession_item_substitutions",
```

Comment:
Can profession item substitions not use `group`?

Reply:
They can to some extent but not like this, I'll just switch it for 2 small cans instead bc it seems pretty overkill to add C++ for this

Reply:
Nvm that doesn't work either